### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by kuddlefishbwaa**
### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

### Description

Changing documentation to make easy for ease of access to contributors

### Related issue

- #369

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -114,18 +114,23 @@ When making a contribution, adhering to the [GitHub flow](https://guides.github.
 
    ```bash
    # Clone your fork of the repo into the current directory
-   git clone https://github.com/<your-username>/<repo-name>
+   git clone https://github.com/<your-username>/Scribe-iOS.git
    # Navigate to the newly cloned directory
-   cd <repo-name>
+   cd Scribe-iOS.git
```
Comment (andrewtavis) [inline CONTRIBUTING.md:119]: For this we don't need the .git :)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -114,18 +114,23 @@ When making a contribution, adhering to the [GitHub flow](https://guides.github.
...
+   - `origin` (forked repository)
+   - `upstream` (Scribe-iOS repository)
+
 
 2. If you cloned a while ago, get the latest changes from upstream:
 
    ```bash
-   git checkout <dev-branch>
-   git pull upstream <dev-branch>
+   git checkout main
```
Comment (andrewtavis) [inline CONTRIBUTING.md:132]: These here we can leave as `<dev-branch>` as was done in the [PR we're referencing] :)

**Comment by andrewtavis** [comment]
We should be good once the changes I mentioned above are fixed, @kuddlefishbwaa. There's gonna be one more minor change in the activist repo, so I'm going to wait till that's finished as maybe we'll want to do that over here as well! Appreciate the help here!

**Comment by kuddlefishbwaa** [comment]
Removed the things requested, I'll wait for more information on what else needs to be done! :)

**Comment by kuddlefishbwaa** [comment]
I can start updating with the info from the other pull request as well if you would like. :)

**Comment by andrewtavis** [comment]
Sure thing, @kuddlefishbwaa! Thanks for keeping on top of all this! 😊

**Comment by andrewtavis** [comment]
Hey there @kuddlefishbwaa 👋 We're done updating the [contributing](https://github.com/activist-org/activist/blob/main/CONTRIBUTING.md) and [readme](https://github.com/activist-org/activist/blob/main/README.md) at activist, so we're good to go as far as making the documentation over here equivalent 😊 Thanks for your patience!

**Comment by kuddlefishbwaa** [comment]
Great I will start looking through that to start making changes

**Comment by andrewtavis** [comment]
Thanks, @kuddlefishbwaa! Looking forward to the further changes :)

**Comment by andrewtavis** [comment]
Another thing that would be great to add in here @kuddlefishbwaa would be to add a reference to [commitlint](https://commitlint.io/) to the contributing guidelines in section 4 of the [pull requests section](https://github.com/scribe-org/Scribe-iOS/blob/main/CONTRIBUTING.md#pull-requests-).

**Comment by andrewtavis** [comment]
I just made [a commit](https://github.com/activist-org/activist/commit/2c81ab57d5d98d54a5e3c7452689c7a12c8ecc74) to activist that adds the following:

```markdown
   > [!NOTE]\
   > The following are tools and methods to help you write good commit messages ✨
   >
   > - [commitlint](https://commitlint.io/) helps write [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
   > - Git's [interactive rebase](https://docs.github.com/en/github/getting-started-with-github/about-git-rebase) cleans up commits
```

**Comment by andrewtavis** [comment]
Quick note on this, @kuddlefishbwaa: we can also use add in a quick bit about how the user can use Xcode to clone a repository in the case of iOS. Happy to add it myself, but wanted to send it along so we remember :)

**Comment by andrewtavis** [comment]
Hey @kuddlefishbwaa 👋 Are you planning any other commits here, or should I review? :)

**Comment by kuddlefishbwaa** [comment]
I'm going to check over some more with some teammates about Xcode and see if they have thoughts on making it easier to walk through! 😄

**Comment by andrewtavis** [comment]
Sounds good, @kuddlefishbwaa :) Thank you!

**Comment by kuddlefishbwaa** [comment]
The team thinks that the Xcode steps are forward enough as is. But I did change some links from the copy over of the starting steps in Contributing. If you have comments or examples of how you would like the Xcode to be outlined if you feel that to be necessary. But otherwise we are good to go!

**Comment by andrewtavis** [comment]
Thank you, @kuddlefishbwaa! I'll review this in the coming days :)

**Review by andrewtavis — APPROVED** [review]
Sorry that this review took so long, @kuddlefishbwaa :) Has been a very busy month and a bit for me. Appreciate the work here! I'll go through and make any needed changes and close out the issue 🚀

**Pull request merged**